client/web/src/components/externalServices/ExternalServiceInformation.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react'
+
+import classNames from 'classnames'
+
+import { ExternalServiceKind } from '@sourcegraph/shared/src/graphql-operations'
+import { Icon, Link, LoadingSpinner, Tooltip } from '@sourcegraph/wildcard'
+
+import styles from '../../site-admin/WebhookInformation.module.scss'
+
+interface ExternalServiceInformationProps {
+    /**
+     * Icon to show in the external service "button".
+     */
+    icon: React.ComponentType<React.PropsWithChildren<{ className?: string }>>
+    kind: ExternalServiceKind
+    displayName: string
+    codeHostID: string
+    reposNumber: number
+    syncInProgress: boolean
+}
+
+export const ExternalServiceInformation: FC<ExternalServiceInformationProps> = props => {
+    const { icon, kind, displayName, codeHostID, reposNumber, syncInProgress } = props
+
+    return (
+        <table className={classNames(styles.table, 'table')}>
+            <tbody>
+                <tr>
+                    <th className={styles.tableHeader}>Code host kind</th>
+                    <td>
+                        <Icon inline={true} as={icon} aria-label="Code host logo" className="mr-2" />
+                        {kind}
+                    </td>
+                </tr>
+                <tr>
+                    <th className={styles.tableHeader}>Display name</th>
+                    <td>{displayName}</td>
+                </tr>
+                <tr>
+                    <th className={styles.tableHeader}>Synced repositories</th>
+                    <td>
+                        <Tooltip content="Click to see the list of repositories">
+                            <Link to={`/site-admin/repositories?codeHost=${encodeURIComponent(codeHostID)}`}>
+                                {reposNumber}
+                            </Link>
+                        </Tooltip>
+                        {syncInProgress && (
+                            <>
+                                {' (Code host sync in progress...'}
```

Comment:
Nitpick, but I think "sync" is confusing customers (they think it means cloning/fetching), so maybe this is a good first step towards improving that:

```suggestion
                                {' (Syncing list of repositories from code host...'}
```

(I kinda think we should go over a lot of our "syncing"/"mirroring" language across the product and try to reduce confusion. I even had the idea to rename this "discovery" instead of "sync" to make it clear that we first "discover" repositories and then clone/fetch/sync them)

Reply:
I agree, sync feels kinda short and very inside, like "repo" and other stuff.
I like the "discovery" tbh!